go.mod

```diff
@@ -4,10 +4,9 @@ go 1.17
 
 require (
 	github.com/ElrondNetwork/elastic-indexer-go v1.2.30
-	github.com/ElrondNetwork/elrond-go v1.3.27
-	github.com/ElrondNetwork/elrond-go-core v1.1.18
+	github.com/ElrondNetwork/elrond-go-core v1.1.21-0.20220928110648-e602417daa59
 	github.com/ElrondNetwork/elrond-go-crypto v1.0.1
-	github.com/ElrondNetwork/elrond-go-logger v1.0.7
+	github.com/ElrondNetwork/elrond-go-logger v1.0.8-0.20221003085013-fb326ba1620f
```

Comment:
reminder to use proper tags before merging

Reply:
done